pkgs/coverage/bin/format_coverage.dart

```diff
@@ -239,11 +239,11 @@ Environment parseArgs(List<String> arguments, CoverageOptions defaultOptions) {
     fail('Package spec "${args["package"]}" not found, or not a directory.');
   }
 
-  if (args['in'] == null && defaultOptions.output == null) {
+  if (args['in'] == null && defaultOptions.outputDirectory == null) {
     fail('No input files given.');
   }
-  final inputPath =
-      args['in'] as String? ?? '${defaultOptions.output}/coverage.json';
+  final inputPath = args['in'] as String? ??
+      '${defaultOptions.outputDirectory}/coverage.json';
   final input = p.absolute(p.normalize(inputPath));
```

Comment:
Same as above. Try `p.normalize((args['in'] as String?) ?? p.absolute(defaultOptions.outputDirectory, 'coverage.json'))`.